src/renderer/components/atoms/text/text.tsx

```diff
@@ -0,0 +1,16 @@
+import type { TextProps } from './types';
+import { resolveElement, buildStylesFor, returnTextColor } from './helper';
+
+const Text = ({ as, variant, useColor, className, children }: TextProps) => {
+  const Element = resolveElement(as);
+  const styles = buildStylesFor(variant || '', as);
+  const TextColor = returnTextColor(as, useColor || '');
```

Comment:
why these `''` values? cann't they just fallback to `undefined` if the prop is not defined?

Reply:
thinking if we don't pass that prop use a default value, I changed it to default values 'normal' for `variant`, and 'primary' for `color` prop